src/main/java/htsjdk/variant/variantcontext/GenotypeBuilder.java

```diff
@@ -182,17 +188,36 @@ public final void reset(final boolean keepSampleName) {
      * Create a new Genotype object using the values set in this builder.
      *
      * After creation the values in this builder can be modified and more Genotypes
-     * created, althrough the contents of array values like PL should never be modified
-     * inline as they are not copied for efficiency reasons.
+     * created
+     *
+     * Note: if attributes are added via this builder after a call to make(), the new Genotype will be modified
      *
      * @return a newly minted Genotype object with values provided from this builder
      */
     public Genotype make() {
-        final Map<String, Object> ea = extendedAttributes == null ? NO_ATTRIBUTES : extendedAttributes;
+        final Map<String, Object> ea = (extendedAttributes == null) ? NO_ATTRIBUTES : extendedAttributes;
         return new FastGenotype(sampleName, alleles, isPhased, GQ, DP, AD, PL, filters, ea);
     }
 
     /**
+     * Create a new Genotype object using the values set in this builder, and perform a
+     * shallow copy of reference types to allow safer re-use of this builder
+     *
+     * After creation the values in this builder can be modified and more Genotypes
+     * created
+     * inline as they are not copied for efficiency reasons.
```

Comment:
Here the "inline as they are not copied for efficiency reasons." should be deleted, since it's not true for this method.